src/@utils/accessDetailsAndPricing.ts

```diff
@@ -191,6 +191,30 @@ export async function getOrderPriceAndFees(
   } catch (error) {
     const message = getErrorMessage(JSON.parse(error.message))
     LoggerInstance.error('[Initialize Provider] Error:', message)
+
+    // Customize error message for accountId non included in allow list
+    if (
+      message.includes(
+        'ConsumableCodes.CREDENTIAL_NOT_IN_ALLOW_LIST' || 'denied with code: 3'
+      )
+    ) {
```

Comment:
I had to look for a substring since the provider is only outputting a generic 400 error. Looking at the provider code, I think the correct error should output `denied with code:3`, but there might be an error with the enum since instead of the error code we are getting the enum itself `ConsumableCodes.CREDENTIAL_NOT_IN_ALLOW_LIST`.
See https://github.com/oceanprotocol/provider/blob/main/ocean_provider/utils/consumable.py and https://github.com/oceanprotocol/provider/blob/176564b67dd505dbdb0e4614794e24d50b283fb9/ocean_provider/utils/credentials.py#L43